x/ethbridge/types/querytypes.go

```diff
@@ -0,0 +1,46 @@
+package types

```

Comment:
consider renaming to `querier.go` as that's generally the purpose for the structs defined here

Reply:
fixed: https://github.com/musnit/peggy/commit/586e80686df398a42e305fc14b58f0ccacb81697